File: android/src/main/kotlin/com/binhit/flutter_open_browser/model/SettingModel.kt
package com.binhit.flutter_open_browser.model

class SettingModel {
    companion object {
        const val SHOW_TOAST = 1
        const val SHOW_DIALOG = -1
    }

    var url: String? = ""
    var intCaseShowError: Int = SHOW_TOAST
    var messageErrorShow: String? = ""
}

File: lib/setting_model.dart
class SettingModel {
  static const int SHOW_TOAST = 1;
  static const int SHOW_DIALOG = -1;
  String url = "";
  int intCaseShowError = SHOW_TOAST;
  String messageErrorShow = "";

  SettingModel({this.url, this.intCaseShowError, this.messageErrorShow});
}
